torch/_export/passes/lift_constants_pass.py

```diff
@@ -90,7 +90,8 @@ def get_constant_fqn(node: torch.fx.Node, constant_name: str) -> str:
     # The FQN of the constant tensor in the state dict should
     # correspond to the module where the constant tensor was
     # originally used.
-    parent_fqn = list(node.meta["nn_module_stack"].values())[-1][0]
+    values = list(node.meta["nn_module_stack"].values())
```

Comment:
I had discussion with @angelayi and @SherlockNoMad for this issue. I think this happens when the tensor constant is not correctly lifted. It will only be lifted if it is converted to a `GetAttr`. I have an example implementation [here](https://github.com/pytorch/pytorch/pull/128442/files#diff-43f47a5a52902847cfa2599feb12166269bdf8fc2ddea670f29e34ef29aec391R318).